spec/declaration.dd

```diff
@@ -347,9 +347,12 @@ $(GNAME AliasDeclarationX):
     $(I AliasDeclarationX) $(D ,) $(GLINK AliasDeclarationY)
 
 $(GNAME AliasDeclarationY):
-    $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK StorageClasses)$(OPT) $(GLINK Type)
-    $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK2 expression, FunctionLiteral)
-    $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK StorageClasses)$(OPT) $(GLINK BasicType) $(GLINK2 function, Parameters) $(GLINK2 function, MemberFunctionAttributes)$(OPT)
+    $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK2 AliasRHS)
+$(GNAME AliasRHS):
+    $(GLINK StorageClasses)$(OPT) $(GLINK Type)
+    $(GLINK2 expression, FunctionLiteral)
+    $(GLINK StorageClasses)$(OPT) $(GLINK BasicType) $(GLINK2 function, Parameters)
+        $(GLINK2 function, MemberFunctionAttributes)$(OPT)
```

Comment:
Because of this new line + identation the [html produced](http://dtest.dlang.io/artifact/website-fa0867dff9d323abeb86cf7617f777e93e253d7c-407ccd88ba35113f87ce2b3185aa0c57/web/spec/declaration.html#alias) does not good, cant you just keep this opt rule on line 354 ?

Reply:
ok